src/app/components/MediaLoader/index.stories.tsx

```diff
@@ -26,18 +28,33 @@ const Component = ({ service, pageType, blocks }: Props) => (
     pathname=""
     service={service}
   >
-    <MediaLoaderComponent blocks={blocks} />
+    <MediaLoaderComponent blocks={blocks} forceStage={experimentStage} />
   </RequestContextProvider>
 );
 
 export default {
   title: 'Components/MediaLoader',
   Component,
+  argTypes: {
+    experimentStage: {
+      options: [Stages.STAGE_2, Stages.STAGE_3],
```

Comment:
Why don't we add all stages here? Also, could it be a dropdown with a default option selected?